atum/src/main/scala/za/co/absa/atum/AtumImplicits.scala

```diff
@@ -94,14 +94,31 @@ trait AtumImplicitsBase {
      */
     def enableControlMeasuresTracking(sourceInfoFilePath: Option[String],
                                       destinationInfoFilePath: Option[String]): SparkSession = {
+      enableControlMeasuresTracking(sourceInfoFilePath, destinationInfoFilePath, DefaultBuildProperties)
+    }
+
+    /**
+     * Enable control measurements tracking on HDFS | S3 (using Hadoop FS API).
+     * Input and output info file names will be inferred automatically based on data source and destination paths if not provided
+     *
+     * Example info file path name: "data/input/wikidata.csv.info" or "s3://bucket1/folder1/wikidata.csv.info"
+     *
+     * @param sourceInfoFilePath      Pathname to a json-formatted info file containing control measurements
+     * @param destinationInfoFilePath Pathname to save the control measurement results to
+     */
+    def enableControlMeasuresTracking(sourceInfoFilePath: Option[String],
```

Comment:
Here, this method is now part of public API, but the tests don't cover it if I am not mistaken.

I suppose extending tests in `HdfsInfoIntegrationSuite` (with a case where this method with custom (non-default) BuildProperties is used) would not be too much effort, but it would nicely support the use-case.

Reply:
> And after the discussion
>
> - will add new unit tests in next commit
> - will add new comments

Yes, tests are to come after we discuss.

Reply:
Cool. I like the direction where it is heading.

Reply:
It is in the integration suite now